torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -483,7 +483,12 @@ ExprHandle TensorExprKernel::getVarForShape(const c10::ShapeSymbol& ss) {
   if (it == shapeSymbolToVar_.end()) {
     VarHandle var("ss" + std::to_string(-value), kLong);
     shapeSymbolToVar_.emplace(value, var);
+#if (defined(__GNUC__) && __GNUC__ >= 13) || \
```

Comment:
Rather than repeat the same condition over and over, please define `C10_COMPILER_SUPPORTS_IMPLICIT_MOVE` or something of the nature

Reply:
C10_RETURN_MOVE_IF_OLD_COMPILER was added.